src/main.py

```diff
@@ -123,6 +124,11 @@ def run_plan_bot(
 
     plans = pure_plans + plan_clusters
 
+    with open(VERBATIMS_FILE) as json_file:
+        verbatims = json.load(json_file)
+
+    plans = plans + verbatims
```

Comment:
🤔 I think this will break something... but I guess I'm not sure...

What's the advantage of combining these here? They seem like fairly different entities to me

Reply:
They are.   I had originally planned to have them be a separate entity and pass them into `process_post` as such.   But I changed my mind, as I'll describe in a more general comment.  :D